cmd/mt-whisper-importer-reader/main.go

```diff
@@ -128,6 +159,13 @@ func main() {
 		defer pos.Close()
 	}
 
+	if *statsEnabled {
+		log.Infoln("graphite stats enabled")
+		stats.NewGraphite(*statsPrefix, *statsAddr, *statsInterval, *statsBufferSize)
+	} else {
```

Comment:
might want to also log something if stats are not disabled, because probably the situation that somebody wants stats and is confused about why they do not get produced is more likely than the other way around.

Reply:
:+1: